File: 32K Kursach/testing LibreTranslate/a.py
from libretranslatepy import LibreTranslateAPI

a='For this God is our God for ever and ever: he will be our guide even unto death. (Psalms 48:14)'

lt = LibreTranslateAPI("https://libretranslate.com/")

print(lt.translate(a, "en", "uk"))
# LibreTranslate es impresionante!

print(lt.detect(a))
# [{"confidence": 0.6, "language": "en"}]

print(lt.languages())
# [{"code":"en", "name":"English"}, {"code":"es", "name":"Spanish"}]

File: 32K Kursach/testing LibreTranslate/a.js
a =
  "For this God is our God for ever and ever: he will be our guide even unto death. (Psalms 48:14)";

const translate = async function (a) {
  const res = await fetch("https://translate.flossboxin.org.in/translate", {
    method: "POST",
    body: JSON.stringify({
      q: a,
      source: "auto",
      target: "uk",
      format: "text",
      alternatives: 3,
      api_key: "",
    }),
    headers: { "Content-Type": "application/json" },
  });

  console.log(await res.json());
};

translate(a);
